lib/erl_docgen/priv/css/otp_doc.css

```
@@ -31,266 +35,446 @@ a:active       { color: #1c7cd6; text-decoration: none }
-  max-width: 52em;
-  overflow-x: hidden;
+    grid-area: content;
+    max-width: 52em;
+    margin-left: 30px;
+}
+
+@media screen and (max-width: 1200px) {
+    #content {
+        margin-left: 0px;
+    }
+}
+
+@media screen and (max-width: 992px) {
+    #content {
+        max-width: 35em;
+    }
+}
+
+.topbar {
+    grid-area: topbar;
+    display: none;
+}
+
+@media screen and (max-width: 768px) {
+
+    #container {
+        grid-template-columns: 100%;
+        grid-template-areas:
+            "topbar"
+            "leftnav"
+            "content";
+    }
+
+    /* styling for the top bar */
+    .topbar {
+        display: flex;
+        position: relative;
+        top: 0px;
+        padding: 0.5em;
+        width: 100%;
+        box-sizing: border-box;
+        border-bottom: 1px solid #ccc;
+        box-shadow: 0 0.5rem 1rem rgba(0,0,0,0.05),
+                    inset 0 -1px 0 rgba(0,0,0,0.15);
+        text-align: center;
+    }
+    .topbar button {
+        padding: 1 6 1 6;
+        margin: 0;
+        height: auto;
+        display: inline-block;
+        border-radius: .25rem;
+        border: 1px solid transparent;
+        background: transparent;
+    }
+    .topbar-expand:not(.show) > button {
+        /* When we add the "show" class we rotate the button 180deg */
+        transform: rotate(180deg);
+    }
+    .topbar-expand > button {
+        transition: transform 0.35s ease;
+    }
+    .topbar-expand.show-permanent > button {
+        display: none;
+    }
+    .topbar-title {
+        text-align: center;
+        flex-grow: 2;
+    }
+    .topbar-title > h1 {
+        font-size: 1.17em;
+        margin: 0px;
+    }
+
+    /* Break words on mobile */
+    #content {
+        overflow-wrap: break-word;
+        word-wrap: break-word;
+        margin-left: 0px;
+        max-width: 100%;
+    }
+    p {
+        max-width: 100%;
+    }
+    /* The h1 is part of the topbar on mobile */
+    #content h1 {
+        display: none;
+    }
+
+    /* Styling of the navbar when shown/hidden */
+    #leftnav {
+        overflow: hidden;
+    }
+    #leftnav:not(.show) {
+        max-height: 0px;
+        border-bottom: 0px;
+        transition: max-height 0.35s ease;
+    }
+    #leftnav.show {
+        max-height: 60vh;
+        border-bottom: 1px solid #ccc;
+        box-shadow: 0 0.5rem 1rem rgba(0,0,0,0.05),
+                    inset 0 -1px 0 rgba(0,0,0,0.15);
+        transition: max-height 0.35s ease;
+    }
+    
+    .leftnav-tube {
+        max-height: 100%;
+        max-width: 100%;
+        position: unset;
+        border-right: 0px;
+        margin-right: 0px;
+        margin-left: 0px;
+        padding: 0px 15px 0px 15px;
+    }
+    #leftnav.show-permanent {
+        max-height: none;
+    }
+    .show-permanent > .leftnav-tube {
+        overflow: initial;
+        max-height: none;
+        height: inherit;
+    }
+    .hide-mobile {
+        obacity: 0;
```

```suggestion
        opacity: 0;
```
? (haven't tested, just noticed a potential typo)

---

Good catch. Thanks.